src/MLDatasets.jl

```diff
@@ -25,8 +25,10 @@ include("require.jl") # export @require
 @require import DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 @require import ImageShow = "4e3cecfd-b093-5904-9786-8bbb286a6a31"
 @require import Chemfiles = "46823bd8-5fb3-5f92-9aa0-96921f3dd015"
+@require import NPZ = "15e1cf62-19b3-5cfa-8e77-841668bca605"
+@require import Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
```

Comment:
we can just normally import Statistics, it is a standard library with little overhead